README.md

```diff
@@ -92,23 +92,101 @@ From edit mode, many operations are possible:
 
-Finally, when an element has a custom text position, it is annoying if the text position resets itself to some computed value whenever a boundary point or edge is moved. To prevent this, there is the *update text position* menu, which when unchecked, leaves the text position as it is whenever a manipulation is done that is not moving the entire element.
+When right clicking an element, or a boundary point or edge of an element, the context menu will contain options for editing the element. These options include opening the properties and style window, removing the entire element, adding boundary points, if an edge is highlighted, and removing a boundary point if one is highlighted. The *move* menu allows you to move elements on top of others, i.e. change the z-index of the element. The properties and style for the entire keyboard can always be opened from the right click menu while in edit mode.
 
 All actions can be undone using `Ctrl` + `Z`, and redone using `Ctrl` + `Shift` + `Z`.
 
+#### Properties and Style
+
+- **Keyboard Properties**
+  - **Size**  
+On this window, you will be able to change the width and height, in pixels, of the keyboard window. It's more precise than dragging the window borders and corners.
+
+- **Keyboard Style**
+  - **Keyboard**
+    - **Background**  
+The background of the keyboard can be a solid color (chosen by double clicking the colored square on the left of the *Background Color* text) or an image from images/ folder located in the category subdirectory (besides the name, you also need to provide the image format).
+
+  - **Loose Keys**/**Pressed Keys**
+    - **Background**  
+The background of the loose/pressed keys can be a solid color or an image from images/ folder located in the category subdirectory.
+
+    - **Text**  
+Here you can change the font type and color of the loose/pressed keys.
+
+    - **Outline**  
+Here you can opt if you want to show the outline of the loose/pressed keys by ticking the CheckBox. You can also change its color and its width.
+
+  - **MouseSpeedIndicator**
+    - **General**  
+The *Color 1* refers to the color of the mouse speed indicator outer circle as well as the color of the direction indicator when the mouse is moving slowly. The *Color 2* refers to the color of the direction indicator when the mouse is moving quickly. The mouse speed is relative to the *Mouse sensitivity* value in the *Settings*. You can also change its width.
+
+- **Elements Properties**
+  - **Keyboard Key Properties**
+    - **Text**/**Shift Text**  
+Text shown on the element when, respectively, the 'Shift' key isn't/is pressed or 'Caps Lock' isn't/is activated.
+
+    - **Text Position**  
+Text position relative to the keyboard itself. The *Center* button automatically calculates the center of the element.
```

Comment:
To the key itself.

Reply:
Isn’t this position relative to the 0 ; 0 position of the keyboard?